ivy/data_classes/container/experimental/activations.py

```diff
@@ -354,11 +354,9 @@ def static_relu6(
 
         Examples
         --------
-        >>> x = {
-                    a: ivy.array([-3., -2., -1., 0., 1., 2., 3., 4., 5.]),
-                    b: ivy.array([1., 2., 3., 4., 5., 6., 7., 8., 9.])
-                }
-        >>> y = ivy.Container.static_relu6(x)
+        >>> x = ivy.Container(a = ivy.array([-3., -2., -1., 0., 1., 2., 3., 4., 5.]),
+        ...                   b = ivy.array([1., 2., 3., 4., 5., 6., 7., 8., 9.]))
+        >>> y = x.static_relu6()
```

Comment:
the correct way to call static methods is like `ivy.Container.static_relu6(x)` as these are not really like instance methods.